### PR TITLE
create entites form database

### PR DESCRIPTION
**Pull request by ChipZero**
ase check nhe

### CODE REVIEW

File: src/main/java/com/elearningbackend/entity/AnswerBank.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		this.answerContent = answerContent;
+		this.creationDate = creationDate;
+		this.lastUpdateDate = lastUpdateDate;
+		this.creatorUsername = creatorUsername;
+		this.lastUpdaterUsername = lastUpdaterUsername;
+	}
+
+	@Id
+	@GeneratedValue(strategy = IDENTITY)
+
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/AnswerBank.java]: remove this line

File: src/main/java/com/elearningbackend/entity/Lession.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import javax.persistence.TemporalType;
+
+/**
+ * Lession generated by hbm2java
+ */
+@Entity
+@Table(name = "lession", catalog = "e_learning")
+public class Lession implements java.io.Serializable {
+
+	/**
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/Lession.java]: remove these asterisks

File: src/main/java/com/elearningbackend/entity/Lession.java [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+		this.lessionCode = lessionCode;
+		this.creationDate = creationDate;
+		this.lastUpdateDate = lastUpdateDate;
+		this.userAnswer = userAnswer;
+		this.userResults = userResults;
+	}
+
+	@Id
+	@GeneratedValue(strategy = IDENTITY)
+
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/Lession.java]: remove this line

File: src/main/java/com/elearningbackend/entity/Lession.java [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+	@Column(name = "user_answer", length = 100)
+	public String getUserAnswer() {
+		return this.userAnswer;
+	}
+
+	public void setUserAnswer(String userAnswer) {
+		this.userAnswer = userAnswer;
+	}
+
+	@OneToMany(mappedBy = "lession", cascade= CascadeType.ALL)
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/Lession.java]: Change to one to one

File: src/main/java/com/elearningbackend/entity/Lession.java [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+	public Set<UserResult> getUserResults() {
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/Lession.java]: public UserResult getUserResult

File: src/main/java/com/elearningbackend/entity/Lession.java [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+		return this.userResults;
+	}
+
+	public void setUserResults(Set<UserResult> userResults) {
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/Lession.java]: public void setUserResult(UserResult userResult) {

File: src/main/java/com/elearningbackend/entity/QuestionBank.java [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+
+/**
+ * QuestionBank generated by hbm2java
+ */
+@Entity
+@Table(name = "question_bank", catalog = "e_learning")
+public class QuestionBank implements java.io.Serializable {
+
+	/**
+	 *
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/QuestionBank.java]: remove these asterisks

File: src/main/java/com/elearningbackend/entity/Lession.java [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+	 */
+	private static final long serialVersionUID = 1L;
+	private Integer lessionId;
+	private QuestionBank questionBank;
+	private User user;
+	private String lessionCode;
+	private Date creationDate;
+	private Date lastUpdateDate;
+	private String userAnswer;
+	private Set<UserResult> userResults = new HashSet<UserResult>(0);
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/Lession.java]: private UserResult userResult;

File: src/main/java/com/elearningbackend/entity/QuestionBank.java [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+	@Column(name = "last_updater_username")
+	public String getLastUpdaterUsername() {
+		return this.lastUpdaterUsername;
+	}
+
+	public void setLastUpdaterUsername(String lastUpdaterUsername) {
+		this.lastUpdaterUsername = lastUpdaterUsername;
+	}
+
+	@OneToMany(mappedBy = "questionBank", cascade= CascadeType.ALL)
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/QuestionBank.java]: One to one

File: src/main/java/com/elearningbackend/entity/QuestionBank.java [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+	public Set<SystemResult> getSystemResults() {
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/QuestionBank.java]: public SystemResult getSystemResult() {
    return this.systemResult;
}

File: src/main/java/com/elearningbackend/entity/QuestionBank.java [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+	private static final long serialVersionUID = 1L;
+	private String questionCode;
+	private Byte questionType;
+	private String questionContent;
+	private String questionParentCode;
+	private Date creationDate;
+	private Date lastUpdateDate;
+	private String creatorUsername;
+	private String lastUpdaterUsername;
+	private Set<SystemResult> systemResults = new HashSet<SystemResult>(0);
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/QuestionBank.java]: private SystemResult systemResult;

File: src/main/java/com/elearningbackend/entity/QuestionBank.java [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+	public void setLastUpdaterUsername(String lastUpdaterUsername) {
+		this.lastUpdaterUsername = lastUpdaterUsername;
+	}
+
+	@OneToMany(mappedBy = "questionBank", cascade= CascadeType.ALL)
+	public Set<SystemResult> getSystemResults() {
+		return this.systemResults;
+	}
+
+	public void setSystemResults(Set<SystemResult> systemResults) {
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/QuestionBank.java]: Sua cai nay nua

File: src/main/java/com/elearningbackend/entity/QuestionBank.java [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+		this.creationDate = creationDate;
+		this.lastUpdateDate = lastUpdateDate;
+		this.creatorUsername = creatorUsername;
+		this.lastUpdaterUsername = lastUpdaterUsername;
+		this.systemResults = systemResults;
+		this.lessions = lessions;
+	}
+
+	@Id
+
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/QuestionBank.java]: Xoa dong trong

File: src/main/java/com/elearningbackend/entity/AnswerBank.java [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+/**
+ * AnswerBank generated by hbm2java
+ */
+@Entity
+@Table(name = "answer_bank", catalog = "e_learning")
+public class AnswerBank implements java.io.Serializable {
+
+	/**
+	 *
```
Comment (halo93) [inline src/main/java/com/elearningbackend/entity/AnswerBank.java]: remove

**Review by monsterddq — APPROVED** [review]

**Pull request merged**